api/adobe_vendor_id.py

```diff
@@ -63,6 +68,83 @@ def status_handler(self):
         return Response("UP", 200, {"Content-Type": "text/plain"})
 
 
+class DeviceManagementProtocolController(BaseCirculationManagerController):
+    """Implementation of the DRM Device ID Management Protocol.
+
+    The code that does the actual work is in DeviceManagementRequestHandler.
+    """
+    DEVICE_ID_LIST_MEDIA_TYPE = "vnd.librarysimplified/drm-device-id-list"
+    PLAIN_TEXT_HEADERS = {"Content-Type" : "text/plain"}
+    
+    @property
+    def link_template_header(self):
+        """Generate the Link Template that explains how to deregister
+        a specific DRM device ID.
+        """
+        url = url_for("adobe_drm_device", device_id="{id}", _external=True)
+        # The curly brackets in {id} were escaped. Un-escape them to
+        # get a Link Template.
+        url = url.replace("%7Bid%7D", "{id}")
+        return {"Link-Template": '<%s>; rel="item"' % url}
+
+    def _request_handler(self, patron):
+        """Create a DeviceManagementRequestHandler for the appropriate
+        Credential of the given Patron.
+       
+        :return: A DeviceManagementRequestHandler
+        """
+        if not patron:
+            return INVALID_CREDENTIALS.detailed("No authenticated patron")
```

Comment:
Yes, in the sense that all our messages should be localized, but this should never be sent in real usage, let alone displayed to a patron.